README.md

```diff
@@ -9,22 +9,13 @@ The project aims to let users without any coding experience separate tracks with
 
 <details id="CannotOpen">
   <summary>Note for macOS users</summary>
+If the application cannot be launched due to the Mac's security protection feature, try the following:
 
-> Because of the limitation of Apple, Demucs-GUI need some extra configuration to work properly.
->
-> First, we should allow running apps from all of sources. Execute following command in your Terminal (if you do not know where your Terminal.app is, please search your dashboard):
->
-> ```bash
-> sudo spctl --master-disable
-> ```
-> You may need to input your password.
->
-> Then, we need to bypass the notarization (replace the path below to where your Demucs-GUI.app is if you did not install to the default location):
->
-> ```bash
-> sudo xattr -rd com.apple.quarantine /Applications/Demucs-GUI.app
-> ```
-
+1. Right-click on the Demucs-GUI app icon and select "Open".
+1. Click "Open" again in the window that appears as follows.
+<p align="center">
+  <img width=300 src="./mac_open_anyway.png"/>
+</p>
```

Comment:
1. Add empty line before and after contents inside `<details>`
2. Do not use `<img>`. Use `![]()` syntax instead.